looper/cli_pydantic.py

```diff
@@ -43,10 +42,46 @@
     is_registry_path,
     read_looper_config_file,
     read_looper_dotfile,
+    initiate_looper_config,
+    init_generic_pipeline,
+    read_yaml_file,
 )
 
+from typing import List, Tuple
 
-def run_looper(args: TopLevelParser, parser: ArgumentParser):
+
+def opt_attr_pair(name: str) -> Tuple[str, str]:
```

Comment:
can you add a 1-sentence docstring to these functions so I have a little more context as to what you're doing?

Reply:
done